### PR TITLE
Added details about possible value of capabilityStatus property

### PR DESCRIPTION
**Pull request by SumitParikh**

### CODE REVIEW

File: api-reference/v1.0/resources/assignedplan.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ Namespace: microsoft.graph
 
 The **assignedPlans** property of both the [user](user.md) entity and the [organization](organization.md) entity is a collection of **assignedPlan**.
```
Comment (dkershaw10) [inline api-reference/v1.0/resources/assignedplan.md:14]: You need to add the same update to the same beta topic please.

Comment (dkershaw10) [inline api-reference/v1.0/resources/assignedplan.md:14]: You need to add the same update to the same beta topic please.

**Review by dkershaw10 — CHANGES_REQUESTED** [review]
Minor update please.  Otherwise LGTM

**Comment by SumitParikh** [comment]
Thanks Dan.

I have updated the PR to make sure same change is added in beta assignedplan resource.

Let me know once the review is complete. I can mark the PR for merge ready.

Thanks and Regards,
Sumit Parikh, Senior Program Manager
Microsoft Corporation
M: +1 425.241.5440

**Comment by SumitParikh** [comment]
#sign-off

File: api-reference/v1.0/resources/assignedplan.md (hunk 1 above)
Comment (SumitParikh) [reply api-reference/v1.0/resources/assignedplan.md:14]: Done.

Comment (SumitParikh) [reply api-reference/v1.0/resources/assignedplan.md:14]: Done.

**Review by angelgolfer-ms — APPROVED** [review]
Thanks for conforming to guidance!

**Pull request merged**